### ISSUE TITLE
A lot of faces seem to be missing.

### ISSUE THREAD

**Issue by FrozeTracker** [SilentNightSound/GI-Model-Importer-Assets#61]
For both the playable and the NPCs. Also textures seem to be missing.

**Comment by SilentNightSound** [comment]
Yes. Faces can't be modded apart from textures, so normally their meshes aren't included. Textures should be mostly there, but it depends if the person who uploaded remembered to grab them or not. If not, you can follow the texture modding tutorial https://github.com/SilentNightSound/GI-Model-Importer/blob/main/Guides/TextureModdingTutorial.md to get the face textures yourself
